### PR TITLE
adding file on bgame restaurants

### LINKED ISSUE #20: places to eat on the peninsula
- [ ] list item

### PR DESCRIPTION
**Pull request by eichnera**
resolves #20

### CODE REVIEW

File: docs/burlingame.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# bgame restaurants 
+- straights
+- urban bistro
```
Comment (eichnera) [inline docs/burlingame.md:3]: @EmilyUna65 ever been to the peninsula

**Comment by EmilyUna65** [comment]
This looks great Angela!

**Comment by eichnera** [comment]
thanks @EmilyUna65  :)

File: docs/burlingame.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+- roti
+
```
Comment (eichnera) [inline docs/burlingame.md:5]: **place a link for roti here**

File: docs/burlingame.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# bgame restaurants 
+- straights
+- urban bistro
+- roti
```
Comment (eichnera) [inline docs/burlingame.md:4]: (place the link to roti here) http://www.yelp.com/search?find_desc=Roti+Indian+Bistro&find_loc=Burlingame%2C+CA

**Comment by AMZod** [comment]
:sheep:

**Comment by watridge** [comment]
:sheep:

**Pull request merged**